contracts/utils/Imports.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { InterchainAddressTracker } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/InterchainAddressTracker.sol';
+import { InterchainAddressTrackerProxy } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/InterchainAddressTrackerProxy.sol';
```

Comment:
ignore warnings

Reply:
@Foivos @milapsheth why do we need this proxy at all? We should just use a regular Proxy.